packages/expo-updates/ios/EXUpdates/DevLauncherAppController.swift

```diff
@@ -110,7 +110,30 @@ public final class DevLauncherAppController: NSObject, InternalAppControllerInte
       return
     }
 
-    if !UpdatesConfig.canCreateValidConfiguration(mergingOtherDictionary: configuration) {
+    // swiftlint:disable:next identifier_name
+    let updatesConfigurationValidationResult = UpdatesConfig.getUpdatesConfigurationValidationResult(mergingOtherDictionary: configuration)
+    switch updatesConfigurationValidationResult {
+    case .Valid:
+      break
+    case .InvalidNotEnabled:
+      errorBlock(NSError(
+        domain: DevLauncherAppController.ErrorDomain,
+        code: ErrorCode.invalidUpdateURL.rawValue,
+        userInfo: [
+          NSLocalizedDescriptionKey: "Failed to read stored updates: configuration object is not enabled"
+        ]
+      ))
+      return
+    case .InvalidPlistError:
+      errorBlock(NSError(
+        domain: DevLauncherAppController.ErrorDomain,
+        code: ErrorCode.invalidUpdateURL.rawValue,
+        userInfo: [
+          NSLocalizedDescriptionKey: "Failed to read stored updates: invalid base plist"
```

Comment:
small suggestion, mention which plist is invalid (I believe we read from Expo.plist, while most people might assume Info.plist).